easybuild/easyblocks/l/llvm.py

```diff
@@ -858,31 +992,16 @@ def _para_test_step(self, parallel=1):
         change_dir(basedir)
         lib_path = ''
         if self.cfg['build_runtimes']:
-            lib_dir_runtime = self.get_runtime_lib_path(basedir, fail_ok=False)
+            lib_dir_runtime = self.get_runtime_lib_path(basedir)
             lib_path = os.path.join(basedir, lib_dir_runtime)
 
-        # When rpath is enabled, the easybuild rpath wrapper will be used for compiling the tests
-        # A combination of -Werror and the wrapper translating LD_LIBRARY_PATH to -Wl,... flags will results in failing
-        # tests due to -Wunused-command-line-argument
-        # This has shown to be a problem in builds for 18.1.8, but seems it was not necessary for LLVM >= 19
-        # needs more digging into the CMake logic
-        old_cflags = os.getenv('CFLAGS', '')
-        old_cxxflags = os.getenv('CXXFLAGS', '')
-        # TODO: Find a better way to either force the test to use the non wrapped compiler or to pass the flags
-        if build_option('rpath'):
-            setvar('CFLAGS', "%s %s" % (old_cflags, '-Wno-unused-command-line-argument'))
-            setvar('CXXFLAGS', "%s %s" % (old_cxxflags, '-Wno-unused-command-line-argument'))
```

Comment:
This part is removed in favor of using the `LLVM-19.1.7_clang_rpathwrap_test.patch` in the easyconfigs

https://github.com/easybuilders/easybuild-easyconfigs/pull/22517/files#diff-2c19368d9e4d19b9a08b4803e398c65a3f599e810fc8cc265d548b747b028fa7